.github/workflows/actions.yml

```diff
@@ -7,16 +7,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: ['2.5', '2.6', '2.7', '3.0', '3.1']
-        rails: ['5.0', '5.1', '5.2', '6.0', '6.1', '7.0']
+        ruby: ['2.6', '2.7', '3.0', '3.1']
```

Comment:
can add 3.2 too

Reply:
not sure what classic is on, but ideally drop 2.6 🤞

Reply:
indeed, classic [is the only user](https://cs.github.com/?scopeName=All+repos&scope=&q=user%3Azendesk+arsi+path%3AGemfile.lock)

Reply:
🎉